docs/src/physics/turbulence_closures.md

```diff
@@ -66,7 +66,7 @@ Each tracer may have a unique diffusivity components ``\kappa_h`` and ``\kappa_v
 In the turbulence closure proposed by [Lilly62](@cite) and [Smagorinsky63](@cite),
 the subgrid stress associated with unresolved turbulent motions is modeled diffusively via
 ```math
-\tau_{ij} = \nu_e \Sigma_{ij} \, ,
+\tau_{ij} = - \nu_e \Sigma_{ij} \, ,
```

Comment:
```suggestion
\tau_{ij} = - 2 \nu_e \Sigma_{ij} \, ,
```

Reply:
I couldn't find this line in the code yet (although I haven't had much time to look for it yet). I just wanna make sure that the code is programmed that way and that it didn't get absorbed into `ν_e` before making this change in the docs.